.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+
+rvm:
+- 2.2
+- 2.1
+- 2.0.0
+- 1.9.3
+- 1.8.7
+- jruby-19mode
+- rbx-2
```

Comment:
You could put a bash conditional on the after_success script to only run based on the current ruby version matching just one of these - but given that this repo is primarily data, rather than a code library, I think just one ruby version here would be fine